CHANGES/6988.feature

```diff
@@ -0,0 +1 @@
+Implemented actions to be taken when installing into a FIPS environment.
```

Comment:
@bmbouter can you help me find a better description?

Reply:
```suggestion
Install a patched version of Django 2.2. LTS, modified for FIPS compatibility on Red Hat based operating systems.
```

Reply:
Thank you!